git-cliff-core/src/release.rs

```diff
@@ -31,35 +33,40 @@ use serde_json::value::Value;
 #[serde(rename_all(serialize = "camelCase"))]
 pub struct Release<'a> {
 	/// Release version, git tag.
-	pub version:    Option<String>,
+	pub version:           Option<String>,
 	/// git tag's message.
-	pub message:    Option<String>,
+	pub message:           Option<String>,
 	/// Commits made for the release.
 	#[serde(deserialize_with = "commits_to_conventional_commits")]
-	pub commits:    Vec<Commit<'a>>,
+	pub commits:           Vec<Commit<'a>>,
 	/// Commit ID of the tag.
 	#[serde(rename = "commit_id")]
-	pub commit_id:  Option<String>,
+	pub commit_id:         Option<String>,
 	/// Timestamp of the release in seconds, from epoch.
-	pub timestamp:  i64,
+	pub timestamp:         i64,
 	/// Previous release.
-	pub previous:   Option<Box<Release<'a>>>,
+	pub previous:          Option<Box<Release<'a>>>,
 	/// Repository path.
-	pub repository: Option<String>,
+	pub repository:        Option<String>,
+	/// Submodule commits.
+	///
+	/// Maps submodule path to a list of commits.
+	#[serde(rename = "submodule_commits")]
+	pub submodule_commits: Option<HashMap<String, Vec<Commit<'a>>>>,
```

Comment:
Changed to Hashmap because indexmap serde implementation doesn't preserve order anyway and order doesn't matter.

Reply:
removed the Option<> wrapping and used an empty hashmap instead. results in a cleaner API during templating.